aws/table_aws_ec2_instance.go

```diff
@@ -365,19 +372,19 @@ func getEc2Instance(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateD
 	return nil, nil
 }
 
-func getAwsEc2InstanceTurbotData(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
-	plugin.Logger(ctx).Trace("getAwsEc2InstanceTurbotData")
+func getEc2InstanceArn(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```

Comment:
```suggestion
func getEC2InstanceARN(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```

I think we want to capitalize any acronyms as per https://github.com/golang/go/wiki/CodeReviewComments#initialisms